core/src/main/java/org/apache/calcite/sql/dialect/BigQuerySqlDialect.java

```diff
@@ -1977,4 +1980,14 @@ private SqlWriter.Frame getColumnListFrame(SqlWriter writer, SqlCall call) {
     }
     return frame;
   }
+
+  private static void unparseGetBitFunction(SqlWriter writer, SqlCall call, int leftPrec,
+      int rightPrec) {
+    call.operand(0).unparse(writer, leftPrec, rightPrec);
+    writer.print(">> ");
```

Comment:
There is a variable named SHIFTRIGHT can we not use it over here

Reply:
Done